tests-legacy/PrestaShopBundle/Utils/DatabaseCreator.php

```diff
@@ -45,6 +46,7 @@ public static function createTestDB()
         require_once __DIR__ . '/../../../install-dev/init.php';
 
         $install = new Install();
+        $install->setTranslator(Context::getContext()->getTranslatorFromLocale('en'));
```

Comment:
PHP 7.4 make this mandatory?

Reply:
In fact, not only for PHP7.4, but if something failed during the install process, you don't receive any message in CLI because the translator is not loaded. It helps me to understand what was wrong :sweat_smile: